scripts/log_instance_stats.py

```diff
@@ -76,12 +76,14 @@ def daemon(redis_url, redis_key, interval):
             "disk_io": psutil.disk_io_counters()._asdict(),
             "net_io": psutil.net_io_counters()._asdict(),
         }
-        for device, mnt_point, fs_type, fs_opts in psutil.disk_partitions():
+        for device, mnt_point, fs_type, fs_opts, max_file, max_path in psutil.disk_partitions():
```

Comment:
should we add `*other` in case they add more return values in `psutil.disk_partitions()`
`*other` can be used as a fail-safe and any new values will be set in a list

Reply:
I guess it shouldn't hurt to add that in there. @pymonger any thoughts on this?

Reply:
Sounds good. It shouldn't hurt.